src/tensorinfo.hpp

```diff
@@ -38,6 +38,9 @@ class TensorInfo;
 
 using tensor_map_t = std::map<std::string, std::shared_ptr<TensorInfo>>;
 
+enum BytesType { encoded,
```

Comment:
Use `enum class`. This is cpp way of declaring enums, this way you do not bloat ovms namespace with raw/encoded, and you can use `ovms::BytesType::encoded` instead of `ovms::encoded`.

Reply:
We use uppercase convention for enums:
https://github.com/openvinotoolkit/model_server/blob/main/src/pipelinedefinitionstatus.hpp#L30